src/core/util/xlio_stats.h

```diff
@@ -464,7 +489,7 @@ typedef struct sh_mem_t {
      * (possibly recursively), may not be a member of a structure or an element of an array.
      * (However, these uses are permitted by GCC as extensions.)
      */
-    socket_instance_block_t skt_inst_arr[1]; // sockets statistics array
+    alignas(64) socket_instance_block_t skt_inst_arr[1]; // sockets statistics array
```

Comment:
Open question:  where should we place alignas() specifier - at the beginning or at the end? We don't have explicit convention for that.

Reply:
alignas replaced with simple paddings because of over alignment nuances